charts/substra-backend/templates/rbac.yaml

```diff
@@ -20,9 +20,12 @@ rules:
     resources: ["secrets"]
     verbs: ["get", "watch", "list", "create", "update", "patch"]
```

Comment:
Do we actually need "create", "update", "patch" on secrets? It looks like this Role just use read access